lib/matplotlib/axes/_axes.py

```diff
@@ -6869,8 +6878,20 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
           keyword argument. Likewise, power-law normalization (similar
           in effect to gamma correction) can be accomplished with
           `.colors.PowerNorm`.
+        - Numpy 1.15 introduced a 'density' kwarg to ``hist2d``. Even though
+          Numpy 1.15 isn't required, 'density' kwarg is introduced
+          and passed as 'normed' to ``hist2d``.
```

Comment:
@jklymak Do you want me to place this explanation somewhere else? Thanks.

Reply:
This probably doesn’t hurt  I might have put it in the normed kwarg description